m5stack-atom-echo/m5stack-atom-echo.yaml

```diff
@@ -53,6 +52,7 @@ speaker:
 voice_assistant:
   id: va
   microphone: echo_microphone
+  micro_wake_word: mww
```

Comment:
Looks like we need to wait for https://github.com/esphome/home-assistant-voice-pe/pull/122 to be upstreamed?

Reply:
Or maybe could import as a custom component like

https://github.com/esphome/home-assistant-voice-pe/blob/7f64fb52b463be9ecaefee3f256bf723b797dfa5/home-assistant-voice.yaml#L1335-L1350